apps/synapse-portal-framework/src/components/DetailsPage/markdown/useColumnDataMarkdownProps.ts

```diff
@@ -0,0 +1,28 @@
+import { MarkdownSynapseProps } from 'synapse-react-client'
+import { transformStringIntoMarkdownProps } from '../../transformStringIntoMarkdownProps'
+import { useDetailsPageContext } from '../DetailsPageContext'
+
+export function useColumnDataMarkdownProps(
+  columnName: string,
+  injectMarkdown: boolean,
+): MarkdownSynapseProps[] | null {
+  let { value } = useDetailsPageContext(columnName)
+
+  if (!value) {
+    return null
+  }
+
+  // TODO: Can we make this safer by checking if the columnType is a LIST?
+  // The column data may be a list of Markdown props to render, so split on commas
```

Comment:
We should probably do this, but we aren't doing it today so I didn't spend time on it.